### PR TITLE
Modernize the README

### PR DESCRIPTION
**Pull request by mattock**
This PR overhauls the README which was largely obsolete.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,65 +1,60 @@
...
-* Create a xxxx.ovpn config-file with your favorite texteditor and save it in:
-  C:\Program files\OpenVPN\config\. You should NOT use the "log" or "log-append"
-  options as OpenVPN GUI redirect the normal output to a log file itself. 
-  There is a sample config files in the "sample-config" folder. Please
-  refer to the OpenVPN project homepage for more information regarding 
-  creating the configuration file. http://openvpn.net/
+Configuring OpenVPN GUI to start on Windows logon
+*************************************************
+
+Put a short-cut to openvpn-gui.exe in your *Start -> All Program -> StartUp*
```
Comment (selvanair) [inline README.rst]: This README is for the master branch, right? So we can unconditionally talk about all v11 features. That means

Adding to startup can be now done very easily from the settings menu.

File: README.rst [hunk 2]
```diff
@@ -1,65 +1,60 @@
...
+Adding an OpenVPN configuration file
+************************************
 
-* Start the GUI by double-clicking the openvpn-gui-1.0.X.exe file. You need to 
-  be Administrator the first time you run OpenVPN GUI for it to create its 
-  registry keys. After that you don't have to be administrator just to run the 
-  GUI, however OpenVPN requires the user to be administrator to run!
+To launch a VPN connections using OpenVPN GUI you must add an OpenVPN
+configuration file with .ovpn suffix and put it to *C:\\Program
+Files\\OpenVPN\config\\*. Any text editor (e.g. notepad.exe) can be used to
```
Comment (selvanair) [inline README.rst]: - The config could be in C:\Users\username\OpenVPN\config or any of its immediate subdirectories as well
- A config file could be imported via the "Import file" menu  
So the "must add  a config to C:\Progra..." statement could be relaxed.

File: README.rst [hunk 3]
```diff
@@ -1,65 +1,60 @@
...
+create this file. You should NOT use the *log* or *log-append* options in the
```
Comment (selvanair) [inline README.rst]: "should NOT" may be too strong -- its ok to have those in the config but will be just ignored. We now pass --log or --log-append as the first option so all subsequent occurrences of --log are ignored by openvpn.

File: README.rst [hunk 4]
```diff
@@ -1,65 +1,60 @@
...
-will be scanned for .ovpn files, and an icon will be displayed in the taskbar's
-status area.
+When OpenVPN GUI is started your OpenVPN config folder
+(*C:\\Program Files\\OpenVPN\\config*) will be scanned for .ovpn files and the
+OpenVPN GUI icon will appear in the system tray. Each OpenVPN configuration file
+shows up as a separate menu item in the OpenVPN GUI tray, allowing you to
+selectively connect to and disconnect to your VPNs.
 
-If you do not have any openvpn connection running, the config dir will be
+If you do not have any OpenVPN connection running, the config dir will be
```
Comment (selvanair) [inline README.rst]: For v11 the rescan happens even if there are connections are running, so new configs are immediately recognized.

File: README.rst [hunk 5]
```diff
@@ -125,16 +115,16 @@ Disconnect  If a file named "xxx_down.bat" exist in the config folder
...
-All OpenVPN GUI reg-values are located below the following key:
-HKEY_LOCAL_MACHINE\SOFTWARE\OpenVPN-GUI\
+All OpenVPN GUI registry values are located below the
+*HKEY_LOCAL_MACHINE\\SOFTWARE\\OpenVPN-GUI\\* key
 
 The follow keys are used to control the OpenVPN GUI
 
 config_dir
-    configuration file directory, defaults to "C:\Program Files\OpenVPN\config"
+    configuration file directory, defaults to *C:\\Program Files\\OpenVPN\\config*
```
Comment (selvanair) [inline README.rst]: May be change the description "configuration file directory" to "system-wide configuration file directory" ? And/or clarify that we also allow user-specific config directory which is hardwired to OpenVPN\config in user's home

File: README.rst [hunk 6]
```diff
@@ -198,38 +188,23 @@ show_balloon
...
-* Run at the OpenVPN GUI source directory::
-  
-    $ autoreconf -i
-    $ ./configure
-    $ make
+The `OpenVPN cross-compile buildsystem <https://github.com/OpenVPN/openvpn-build>`_ builds OpenVPN GUI along all the
+other OpenVPN dependencies. Instructions and automated scripts for setting up
+the buildsystem are available on the
+`Building OpenVPN using the generic buildsystem <https://community.openvpn.net/openvpn/wiki/BuildingUsingGenericBuildsystem>`_
+page on the OpenVPN community Wiki.
```
Comment (selvanair) [inline README.rst]: Also we could add a reference to Build.rst?

File: README.rst (hunk 3 above)
Comment (mattock) [inline README.rst]: That was the original wording, but I can sure change it.

File: README.rst (hunk 4 above)
Comment (mattock) [inline README.rst]: Yeah, I'll change that.

File: README.rst (hunk 5 above)
Comment (mattock) [inline README.rst]: Indeed, good point.

File: README.rst (hunk 6 above)
Comment (mattock) [inline README.rst]: Definitely. I actually was not aware of that file.

File: README.rst (hunk 1 above)
Comment (mattock) [inline README.rst]: Yes, this is for the master branch. We probably need not touch README in the v10 branch, even though it's quite outdated even there.

**Comment by mattock** [comment]
@selvanair: all the issues you identified have now been fixed. I added openvpn-build instructions to BUILD.rst, and then linked to BUILD.rst from README.rst.

**Comment by selvanair** [comment]
Looks good. Merging.

**Pull request merged**